packages/tools/devtools/devtools-core/src/CommonInterfaces.ts

```diff
@@ -33,7 +33,7 @@ export interface HasContainerKey {
  *
  * @internal
  */
-export type FluidObjectId = string;
+export type FluidObjectId = number;
```

Comment:
If possible, I think I'd actually prefer to keep this a string. Even if we continue to generate numeric values internally, making the JSON interop see this as a string seems simpler. It also lets us keep our custom keys in test cases and such. Alternatively, we could make it `string | number`, but I'm not sure that would add much value since you can implicitly convert numbers to strings.